index.html

```diff
@@ -86,6 +86,5 @@ <h1 class="title">Back-End Tools</h1>
         Django (/ˈdʒæŋɡoʊ/ JANG-goh)[5] is a free and open-source web framework, written in Python, which follows the model-view-template (MVT) architectural pattern.
        </span>
     </div>
-  </div>
```

Comment:
I think this div is needed. Isnt it the one that closes the techos-container div?

Reply:
that is very true. will be fixed on the next commit